Dockerfile

```diff
@@ -50,23 +67,32 @@ RUN \
  && pip3 install kafka-python==1.3.2 \
  && pip3 install bokeh==0.12.4
 
+# Create a directory to
+RUN mkdir ~/.ssh 
+COPY ./config/ssh_config .ssh/config
+
 RUN \
-# Sbt
+# git clone during image build and setting enviroment variables
+ cd ~ \
+ && git clone https://yogeshgo05@github.com/abgoswam/streamai
+
+RUN \
+ #Sbt
  cd ~ \
  && wget https://dl.bintray.com/sbt/native-packages/sbt/${SBT_VERSION}/sbt-${SBT_VERSION}.tgz \
  && tar xvzf sbt-${SBT_VERSION}.tgz \
  && rm sbt-${SBT_VERSION}.tgz \
  && ln -s /root/sbt/bin/sbt /usr/local/bin \
-# Sbt Clean - This seems weird, but it triggers the full Sbt install which involves a lot of external downloads
- && sbt clean clean-files 
-
+ #Sbt Clean - This seems weird, but it triggers the full Sbt install which involves a lot of external downloads
+ && sbt clean clean-files
 
 RUN \
-# Apache Kafka 
+ #Apache Kafka
  cd ~ \
  && wget http://apache.claz.org/kafka/${KAFKA_CLIENT_VERSION}/kafka_${SCALA_MAJOR_VERSION}-${KAFKA_CLIENT_VERSION}.tgz \
  && tar -xvzf kafka_${SCALA_MAJOR_VERSION}-${KAFKA_CLIENT_VERSION}.tgz  \
  && rm kafka_${SCALA_MAJOR_VERSION}-${KAFKA_CLIENT_VERSION}.tgz
 
 
+
```

Comment:
if possible lets put 'streamai/binsc/setup/RUNME_ONCE.sh' in the docker file so that kafka and zookeeper start by default when we spin up new container